src/emm/services/pull_request_service.py

```diff
@@ -32,6 +32,38 @@ def pr_comment(self) -> str:
         return f"* [{self.name}]({self.link})"
 
 
+class PullRequestOption(NamedTuple):
+    """
+    Options for creating a pull request.
+
+    * title: Title of the pull request.
+    * branch: Remote branch of the pull request.
+    * remote_url: URL to pull request in github.
+    * body: Body of the pull request.
+    """
+
+    title: str
+    body: str
+    branch: str
+    remote_url: str
+
+    def option_list(self) -> List[str]:
+        """Convert pull request arguements to arguement lists."""
+        if self.remote_url == DEFAULT_REMOTE:
+            return ["--title", self.title, "--body", self.body]
+
```

Comment:
Note when specify a repo, Github cli cannot automatically calculate commit message with "--fill" option. Therefore we have to specify title each time, I use git to retrieve the commit message.